app-i18n/fbterm/fbterm-1.7_p20190503.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
```

Comment:
If you're updating the URL, you should do it in all ebuilds. I'm not sure which HOMEPAGE packages.gentoo.org and tools like eix use when it has different values (probably that of the first ebuild they find).

Reply:
This is misadvice.
HOMEPAGE in older versions should point to this upstream from whom those versions come from.
Tools show HOMEPAGE from newest ebuild.

Reply:
Good to know. Removed that commit.